### PR TITLE
Add the description for using call stack graph.

### PR DESCRIPTION
**Pull request by weilonge**

### CODE REVIEW

**Comment by weilonge** [comment]
@begeeben Please help to review this. Thank you!

File: README.md [hunk 1]
```diff
@@ -93,11 +93,15 @@ $ ./cli.js parse ./srouce_dir
 
 Automatically add console.log after each functions
 ```sh
-$ ./cli.js autolog ./srouce_dir
+$ ./cli.js autolog ./srouce_dir [-c]
```
Comment (dwi2) [inline README.md]: Typo. I think it should be
```sh
$ ./cli.js autolog ./source_dir [-c]
```
right?

Comment (weilonge) [inline README.md]: This should be a typo. I've updated it. Thank you.

**Comment by dwi2** [comment]
This looks good to me. :+1:

**Pull request merged**